app/views/mail_shared/panel/_proposal_attributes.html.haml

```diff
@@ -4,6 +4,8 @@
   subtext: proposal.public_id }
 
 - proposal.fields_for_display.each_slice(2) do |fields|
+  - field_1_data = !property_to_s(fields[0][1]).empty? ? property_to_s(fields[0][1]) : '—'
+  - field_2_data = !property_to_s(fields[1][1]).empty? ? property_to_s(fields[1][1]) : '—'
```

Comment:
`fields[1]` will sometimes be `nil`, which is why we have the check on L 22. I can help put this into a helper, but I think we could define something in `MailerHelper` that takes care of this and would prevent the `nil` errors we're getting in specs:

```ruby
# MailerHelper

def property_display_value(field)
 if property_to_s(field[1])
    property_to_s(field[1])
  else
    "-"
  end
end
```

and then here where we used to have

`property_to_s(fields[0][1])` and `property_to_s(fields[1][1])`

we can call

`property_display_value(fields[0])` and `property_display_value(fields[1])`